packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -628,3 +689,44 @@ export const waitTextVisible = ({ context, text, isVisible = true }: { text: Pat
         }
     });
 };
+
+const continueOnInvalidPage: ContinueWithInvalidEntriesTest = async ({ context, text, currentPageUrl, nextPageUrl }) => {
+    const button = context.page.getByRole('button', { name: text });
+    await button.scrollIntoViewIfNeeded();
+    await button.click();
+    await expect(context.page.waitForURL(nextPageUrl, {timeout: 1000})).rejects.toThrow("Timeout")
+    await expect(context.page).toHaveURL(currentPageUrl);
+}
+
+/**
+ * Click the next page button when not all options are filled and check that we remain on the same page.
+ *
+ * @param {Object} options - The options for the test.
+ * @param {string} options.text - The text of the button to click.
+ * @param {string} options.currentPageUrl - The URL of the current page.
+ * @param {string} options.nextPageUrl - The URL of the page that the button would normally take us too.
+ */
+export const continueOnEmptyPage: ContinueWithInvalidEntriesTest = ({ context, text, currentPageUrl, nextPageUrl }) => {
+    test(`Clicking ${text} when options are empty should keep you on ${currentPageUrl} - ${getTestCounter(context, `${text} - ${currentPageUrl} - ${nextPageUrl}`)}`, async () => {
+        await continueOnInvalidPage({ context, text, currentPageUrl, nextPageUrl });
+    });
+};
+
+/**
+ * Click the next page button when an age under 18 is clicked, verifies that the appropriate popup appears, and close it.
+ *
+ * @param {Object} options - The options for the test.
+ * @param {string} options.text - The text of the button to click.
+ * @param {string} options.currentPageUrl - The URL of the current page.
+ * @param {string} options.nextPageUrl - The URL of the page that the button would normally take us too.
+ */
+export const continueWithUnder18Age: ContinueWithInvalidEntriesTest = ({ context, text, currentPageUrl, nextPageUrl }) => {
```

Comment:
This sounds very specific. It is actually a goto next page with a popup that appears. It could happen for any kind of reason other than the under 18.